### PR TITLE
Added 0800 support on br-phone-number

### PR DESCRIPTION
**Pull request by neokyox**
Added 0800 support when typed in the beggining of the phone number, formated as 0800-123-1234 or 0800-123-123, since both are supported.

### CODE REVIEW

File: src/br/phone/br-phone.test.js [hunk 1]
```diff
@@ -47,7 +56,7 @@ describe('ui-br-phone-number', function() {
 			{value:'@', viewValue:'', modelValue:''},
 			{value:'2-', viewValue:'(2', modelValue:'2'},
 			{value:'23a', viewValue:'(23', modelValue:'23'},
-			{value:'23_34', viewValue:'(23) 34', modelValue:'2334'},
+			{value:'23_34', viewValue:'(23) 34nu', modelValue:'2334'},
```
Comment (assisrafael) [inline src/br/phone/br-phone.test.js]: Seems that the change of this line is breaking the tests. Can you fix it?

**Comment by assisrafael** [comment]
Thank you for submitting this pull request.
However I cannot merge it until all tests pass.

**Comment by neokyox** [comment]
I'm working on it. 0800 numbers are losing the first 0 on tests also

**Comment by neokyox** [comment]
All good now. Unless you want me to do something else

**Comment by assisrafael** [comment]
Only one more thing: squash all commits into a single one.

**Comment by neokyox** [comment]
Done. Thanks for the patience :)

**Pull request merged**